### PR TITLE
Fix about list styles

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

Fixes styles in list about page

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/charterafrica/src/components/PageDescription/PageDescription.js [hunk 1]
```diff
@@ -37,6 +37,12 @@ const PageDescription = React.forwardRef(function PageDescription(props, ref) {
                 ...theme.typography.subheading,
               },
             },
+            "& li": {
+              ...theme.typography.p1,
```
Comment (kilemensi) [inline apps/charterafrica/src/components/PageDescription/PageDescription.js:41]: Doesn’t `font: inherit` work?

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: apps/charterafrica/src/components/PageDescription/PageDescription.js (hunk 1 above)
Comment (koechkevin) [reply apps/charterafrica/src/components/PageDescription/PageDescription.js:41]: Does,t seem to work @kilemensi

**Review by kelvinkipruto — APPROVED** [review]

**Review by kilemensi — APPROVED** [review]
👍🏽

**Pull request merged**